Branch-SDK/Branch-SDK/BNCServerInterface.m

```diff
@@ -256,9 +265,18 @@ - (BNCServerResponse *)processServerResponse:(NSURLResponse *)response data:(NSD
         [self.preferenceHelper log:FILE_NAME line:LINE_NUM message:@"returned = %@", serverResponse];
     }
     
+    [self collectInstrumentationMetrics];
     return serverResponse;
 }
 
+- (void) collectInstrumentationMetrics {
+    // multiplying by negative because startTime happened in the past
+    NSTimeInterval elapsedTime = [startTime timeIntervalSinceNow] * -1000.0;
+    NSString *lastRoundTripTime = [[NSNumber numberWithDouble:floor(elapsedTime)] stringValue];
+    NSString * lrttKey = [NSString stringWithFormat:@"%@-lrtt", requestEndpoint];
```

Comment:
ON Android there was  an implementation mistake and hence `lrtt ` was sending only the connection open time. To avoid the confusion with previous versions of sdk , we are adding the fix with new name `brtt`  ( Branch Round Trip Time)